apps/mobile/src/state/chat/atoms/selectOtherSideDataAtom.ts

```diff
@@ -10,6 +10,7 @@ export default function selectOtherSideDataAtom(
   chatAtom: Atom<Chat>
 ): Atom<UserNameAndAvatar> {
   return selectAtom(chatAtom, (chat) => {
+    console.log(`Chat: ${JSON.stringify(chat, null, 2)}`)
```

Comment:
This is probably leftover from debugging. Remove or change log to be more descriptive